### DISCUSSION TITLE
Which commmand to use to edit multi valued attribute

### DISCUSSION THREAD

**Discussion by quratulain-ali** [eclipse-glsp/glsp#1261] [General] [answered]
Which model server command to use to edit multi-valued properties? When I use the AddCommand it adds the element to multivalued property.

**Comment by planger** [comment]
Hi @quratulain-ali,
thank you for your question!
I think this better fits into the [EMF Cloud forums](https://github.com/eclipse-emfcloud/emfcloud/discussions). I'll transfer this thread.

**Reply by planger** [reply, marked as answer]
Oh, as it turns out, I can't move across Github organizations. @quratulain-ali May I ask you to open this question again in https://github.com/eclipse-emfcloud/emfcloud/discussions? Thank you!

**Reply by quratulain-ali** [reply]
Cool thanks!
